backend/open_webui/models/skills.py

```
@@ -0,0 +1,339 @@
+                resource_type="skill",
+                resource_id=skill.id,
+                permission=permission,
+                user_group_ids=user_group_ids,
+                db=db,
+            )
+        ]
+
+    def search_skills(
+        self,
+        user_id: str,
+        filter: dict = {},
```

Ruff (0.15.0)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)